spog/ui/src/pages/search.rs

```diff
@@ -25,6 +28,17 @@ pub enum TabIndex {
     Packages,
 }
 
+impl std::fmt::Display for TabIndex {
```

Comment:
Just a comment, there is a crate call `strum` which can do that for you. But it's ok to merge this anyway.

Reply:
@ctron Thanks for the advice. I didn't realize we are using `strum`. I've just applied it on the last commit! Thanks